pkgs/development/libraries/audio/vamp/default.nix

```diff
@@ -1,16 +1,19 @@
 # set VAMP_PATH ?
 # plugins availible on sourceforge and http://www.vamp-plugins.org/download.html (various licenses)
 
-{ stdenv, fetchurl, pkgconfig, libsndfile }:
+{ stdenv, fetchFromGitHub, pkgconfig, libsndfile }:
 
 rec {
 
   vampSDK = stdenv.mkDerivation {
```

Comment:
If you make this ```vampSDK = stdenv.mkDerivation rec {``` you can use version properly below.

Reply:
I'm sorry, I don't get what you are saying here.

Reply:
I have just pushed a commit to your branch.